Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -49,8 +50,9 @@ extension KeyboardShortcuts {
 			}
 		}
 
-		public required init(for name: Name) {
+		public required init(for name: Name, onChange: ((_ shortcut: Shortcut?) -> Void)? = nil) {
```

Comment:
Can you add a documentation comment for `onChange`? Also in the other initializer.

Also mention the use-case for this (your use-case). Users don’t always realize why they would need a certain feature, and presenting some use-cases makes it clearer.